### DISCUSSION TITLE
Java implementation

### DISCUSSION THREAD

**Discussion by bufdev** [connectrpc/.github#11] [Polls]
Would you or your company use a Connect implementation in Java? If yes, please leave a comment and tell us more about what you'd like (especially if you need integration with a particular framework).

**Comment by alexanderaboutanos** [comment]
please oh please! :)

**Reply by cliffbdf** [reply]
I am somewhat shocked that a Java version was not first, given the size, stability, and value of the Java ecosystem, especially in the B2B world.

**Comment by evoshawkins** [comment]
Yes, server only.

**Comment by vladimir-ikryanov** [comment]
Yes, server only.

**Comment by mbazos** [comment]
Please start with the server, client can come later.

**Comment by kamalhm** [comment]
Yes please

**Comment by lomeshpatel** [comment]
It would be great if support for Spring and Quarkus was also included.

**Comment by igor-vovk** [comment]
I started working on [connect-rpc-java](https://github.com/igor-vovk/connect-rpc-java) library, which is a transfer of [connect-rpc-scala](https://github.com/igor-vovk/connect-rpc-scala) library. Everything looks promising so far. In the first release I'm planning to include Connect RPC server support on top of Netty (no client, no streaming) and GRPC Transcoding. You can follow it if you wish to know about the updates.

**Comment by sbussard-inversemetric** [comment]
Please add java server support
